test/system/preTestScript.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+CLUSTER_NAME=${1:-null}
```

Comment:
The script is only for jarvis environments and jarvis client should be installed. Could you add a comment stating the same?
This will be modified in future for handling cloud deployements too right?

Reply:
done, yes it might be modified , atleast we dont need jarvis commands there.